app/models/checkout.rb

```diff
@@ -30,16 +34,28 @@ def recalled?
     recalled_date.present?
   end
 
-  def patron_key
-    fields['patron']['key']
+  def claims_returned_date
+    Time.zone.parse(fields['claimsReturnedDate']) if fields['claimsReturnedDate']
+  end
```

Comment:
This could also potentially be `private` I think.

Reply:
yes, this can be private 👍 thanks for catching it. I will update it and remove the related specs.